frontend/src/views/Collection/components/ActionButtons/style.ts

```diff
@@ -1,10 +1,12 @@
 import { PT_GRID_SIZE_PX } from "src/components/common/theme";
 import styled from "styled-components";
 
-export const StyledDiv = styled.div`
+export const Wrapper = styled.div`
+  grid-column: 6 / span 3;
+  justify-self: end;
   display: flex;
   flex-direction: row;
   & > :not(:last-child) {
-    margin-right: ${PT_GRID_SIZE_PX}px;
+    margin-right: ${PT_GRID_SIZE_PX * 2}px;
```

Comment:
Since it's now used on the upper right corner, the spec has changed from 8px to 16px